main.go

```diff
@@ -26,7 +26,6 @@ import (
 	// +kubebuilder:scaffold:imports
 	asocontainerservicev1api20210501 "github.com/Azure/azure-service-operator/v2/api/containerservice/v1api20210501"
 	asocontainerservicev1api20230201 "github.com/Azure/azure-service-operator/v2/api/containerservice/v1api20230201"
-	asocontainerservicev1api20230202preview "github.com/Azure/azure-service-operator/v2/api/containerservice/v1api20230202preview"
```

Comment:
This version was removed in the new ASO version: https://github.com/Azure/azure-service-operator/releases/tag/v2.9.0